wordpress-mobile/WordPress-iOS
# Quick Start for Existing Users: Update Quick Start complete notification copy

Part of #18389

WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/503

## Description
- Updated WPKit so that the Quick Start complete notification reads "Next Steps" instead of "Quick Start"

## Notes
- Once this PR is approved I will release a new beta for WPKit and point to that new beta

## How to test
1. Trigger Quick Start from the debug menu
2. Go through all tours
3. ✅ Verify that that the Notification reads "You've finished the Next Steps checklist..." instead of "You've finished the Quick Start checklist..."

![IMG_3D14FC3503FC-1](https://user-images.githubusercontent.com/6711616/167661895-605043a8-6dc9-4f2b-a4e3-c4bd17655217.jpeg) | ![IMG_688C2455FCDF-1](https://user-images.githubusercontent.com/6711616/167662194-8ee6c8dc-00d1-4ffe-a611-c4524ca34638.jpeg)
-- | --

## Regression Notes
1. Potential unintended areas of impact
n/a

4. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

5. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
